.clang-tidy

```diff
@@ -16,8 +16,8 @@ HeaderFilterRegex: '.*'
 ExcludeHeaderFilterRegex: 'include/lmdb.h|Grammar.h'
 UseColor: true
 FormatStyle: 'file'
-ExtraArgs: ['-std=c++20']
+ExtraArgs: ['-std=c++17']
```

Comment:
Otherwise we get lints for issues that require using C++ 20 and we are still using C++ 17 with GCC.